src/lib/autoinstall/widgets/storage/lvm_page.rb

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage"
+require "cwm/page"
+require "autoinstall/widgets/storage/add_children_button"
+require "autoinstall/widgets/storage/vg_device"
+require "autoinstall/widgets/storage/vg_extent_size"
+require "autoinstall/widgets/storage/md_level"
+require "autoinstall/widgets/storage/chunk_size"
+require "autoinstall/widgets/storage/parity_algorithm"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # This page allows to edit a `drive` section representing an LVM
+      class LvmPage < ::CWM::Page
+        # Constructor
+        #
+        # @param controller [Y2Autoinstallation::StorageController] UI controller
+        # @param section [Y2Storage::AutoinstProfile::DriveSection] Drive section corresponding
+        #   to a RAID
+        def initialize(controller, section)
+          textdomain "autoinst"
+          @controller = controller
+          @section = section
+          super()
+          self.widget_id = "raid_page:#{section.object_id}"
+          self.handle_all_events = true
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          format(_("LVM %{device}"), device: section.device)
+        end
+
+        # @macro seeCustomWidget
+        def contents
+          VBox(
+            Left(Heading(_("LVM"))),
+            VBox(
+              Left(vg_device_widget),
+              Left(vg_pesize_widget),
+              VStretch()
+            ),
+            HBox(
+              HStretch(),
+              AddChildrenButton.new(controller, section)
+            )
+          )
+        end
+
+        # @macro seeAbstractWidget
+        def init
+          vg_device_widget.items = controller.lvm_devices
```

Comment:
It's still not clear to me if we should use this logic or the inverse one (ie, completing the lvm_group fields of the partitions based on the existing CT_LVM sections).

Reply:
Ah! I forgot that. Sorry.

Reply:
Done! See https://github.com/yast/yast-autoinstallation/pull/596/commits/7e1ca9745eee9e98aad74c2fa8198ac9f904943b